utility/application-util.js

```diff
@@ -0,0 +1,31 @@
+const {IN_PROGRESS} = require("../constants/application-constants");
+const {HistoryEventBuilder} = require("../domain/history-event");
+const ERROR = require("../constants/error-constants");
+const {UpdateApplicationStateEvent} = require("../crdc-datahub-database-drivers/domain/log-events");
+
+
+async function updateApplication(applicationCollection, application, prevStatus, userID) {
+    if (prevStatus !== IN_PROGRESS) {
```

Comment:
If we share this function, we should include parameter instead of 'IN_PROGRESS'.